### ISSUE TITLE
Add serve script for Statamic

### ISSUE THREAD

**Issue by svpernova09** [laravel/homestead#433]
http://manuals.gravitydept.com/develop/environment-setup/homestead

```
For Statamic

Works great with one modification for the Statamic admin.

Edit /Users/brendan/work/repos/homestead/scripts/serve-laravel.sh and find this code:

location / {
    # GravDept: add this line for Statamic
    rewrite ^/admin.php.*$ /admin.php;
    # GravDept: end

    try_files \$uri \$uri/ /index.php?\$query_string;
}
Remember to run homestead provision afterward.
```

**Comment by brendanfalkowski** [comment]
Note: that might break some other frameworks/code but I really only use Laravel and Statamic on Homestead so it's ok for me.

**Comment by svpernova09** [comment]
@brendanfalkowski Sure, but if we add a Statamic server script you would just add an option to your site to specify `statamic` and it would use the right server script to create your host

**Comment by brendanfalkowski** [comment]
@svpernova09 Hmm, I guess that'd be cool. I honestly didn't look too deeply at what the scripts do, but now it makes sense. Would you specify that in Homestead.yaml? I hadn't seen an example like that (way back when).

**Comment by svpernova09** [comment]
You would define your Statamic sites with a type key:

```
sites:
    - map: statamic.app
      to: "/home/vagrant/statamic/public"
      type: "statamic"
```

**Comment by brendanfalkowski** [comment]
Ah ok, I thought that was just for folder mapping, but if that works it'll be nice not to have to maintain the Statamic mods across Homestead updates.

**Comment by svpernova09** [comment]
Resolved by #436
